### PR TITLE
2017-10-06-WeChat PC Impactor

### PR DESCRIPTION
**Pull request by h4dex**
微信电脑版 HOOK 的一些研究 以及代码分享！ 
 
感谢精易论坛坛友(xdssoft,恨不能遗忘,gh0st少主)分享的开源代码参考！

### CODE REVIEW

File: _posts/2017-10-06-WeChat PC Impactor [hunk 1]
```diff
@@ -0,0 +1,488 @@
+## 关于对微信PC版Hook的一点研究分享：
```
Comment (lijiarui) [inline _posts/2017-10-06-WeChat PC Impactor]:
```
---
 layout: post
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
Please add the title for the article like other post.

File: _posts/2017-10-06-WeChat PC Impactor [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+
+
+
+感谢 易语言精易论坛坛友(xdssoft,恨不能遗忘,gh0st少主)分享的开源代码参考！ 
+
+因为易语言代码有的朋友们可能看起来怪怪的 如果没有接触过它，并不是那么通俗易懂。所以改成了cpp代码供审阅.  由于书写仓促 错误的部分希望大家指正，小弟在此多谢！
+
```
Comment (lijiarui) [inline _posts/2017-10-06-WeChat PC Impactor]: Also, in order to just show an abstract in the blog home, please add `<!--more-->` after the abstract or the complete content will show on the blog home.

Thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Review by huan — CHANGES_REQUESTED** [review]
In order to keep the style consistent for through all the blogs, please follow the reviews from @lijiarui. 

I have two requests too, please:
1. rename `2017-10-06-WeChat PC Impactor` to `2017-10-06-wechat-pc-impactor`
1. rename `WechatWindowsImpactorSrc.rar` to `wechat-windows-impactor-src.rar` (it would be better if you could compress it to .zip because it's more standard)

That's because we should keep all filenames & url as lowercase, and use `-` to connect words instead of space.

After you finished all the change requests, we will be able to merge this great PR to publish on our Chatie blog.

Thank you very much!

**Comment by huan** [comment]
And could you please select a good image for your blog? It would be fantastic if a blog has a beautiful face image.

**Comment by lijiarui** [comment]
Just one more thing:   rename 2017-10-06-WeChat-PC-Impactor to 2017-10-06-wechat-pc-impactor

Pay  attention to lowcase, then I will merge this.

Thanks!

**Review by huan — APPROVED** [review]
Looks awesome to me, Thanks!

**Pull request merged**